### PR TITLE
Show throughput in megabits per second

### PR DESCRIPTION
**Pull request by siggisim**
Use megabits per second instead of megabytes per second which is more common for networking

---

**Version bump**: TODO

### CODE REVIEW

File: enterprise/app/trends/trends.tsx [hunk 1]
```diff
@@ -213,35 +215,41 @@ export default class TrendsComponent extends React.Component {
...
                 }
                 extractLabel={this.formatShortDate}
                 formatHoverLabel={this.formatLongDate}
-                formatHoverValue={(value) => (value || 0).toFixed(2) + " MB downloaded"}
-                formatSecondaryHoverValue={(value) => (value || 0).toFixed(2) + " MB / sec"}
-                name="MB downloaded"
-                secondaryName="MB / sec download throughput"
+                formatHoverValue={(value) => (value || 0).toFixed(2) + " Mb downloaded"}
+                formatSecondaryHoverValue={(value) => (value || 0).toFixed(2) + " Mbps"}
+                name="Mb downloaded"
```
Comment (tylerwilliams) [inline enterprise/app/trends/trends.tsx:230]: Can we keep transfer sizes in Megabytes (MB) and only change throughputs to Mbps? Otherwise lgtm

**Review by tylerwilliams — CHANGES_REQUESTED** [review]

File: app/invocation/invocation_cache_card.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ interface Props {
   model: InvocationModel;
 }
 
+const bitsPerByte = 8;
```
Comment (bduffany) [inline app/invocation/invocation_cache_card.tsx:10]: nit: Do we want to use CAPITALS_WITH_UNDERSCORES for constants?

File: app/invocation/invocation_cache_card.tsx [hunk 3]
```diff
@@ -25,8 +27,8 @@ export default class CacheCardComponent extends React.Component {
           {this.props.model.cacheStats.length && (
             <div className="details">
               {this.props.model.cacheStats.map((cacheStat) => {
-                let downloadThroughput = +cacheStat.downloadThroughputBytesPerSecond / 1000000 || 0;
-                let uploadThroughput = +cacheStat.uploadThroughputBytesPerSecond / 1000000 || 0;
+                let downloadThroughput = bitsPerByte * (+cacheStat.downloadThroughputBytesPerSecond / 1000000) || 0;
```
Comment (bduffany) [inline app/invocation/invocation_cache_card.tsx:30]: nit: `|| 0` can be removed since all types in the expression are `number`

**Review by bduffany — APPROVED** [review]

File: enterprise/app/trends/trends.tsx (hunk 1 above)
Comment (siggisim) [reply enterprise/app/trends/trends.tsx]: Done

File: app/invocation/invocation_cache_card.tsx (hunk 2 above)
Comment (siggisim) [reply app/invocation/invocation_cache_card.tsx]: Done

File: app/invocation/invocation_cache_card.tsx (hunk 3 above)
Comment (siggisim) [reply app/invocation/invocation_cache_card.tsx]: Done

**Review by bduffany — APPROVED** [review]

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Pull request merged**